.github/workflows/python-test.yml

```diff
@@ -87,35 +87,35 @@ jobs:
       - name: Show path of binaries
         shell: bash
         run: |
-          if [ "${{ matrix.version }}" = "15" -a "${{ matrix.os }}" = "windows-latest" ]; then
-            which clang-format
-            which clang-tidy
-          elif [ "${{ matrix.version }}" = "12.0.1"  -a "${{ matrix.os }}" = "ubuntu-latest" ]; then
-            which clang-format-12
-            which clang-tidy-12
-          else
-            which clang-format-${{ matrix.version }}
-            which clang-tidy-${{ matrix.version }}
+          if [ "${{ matrix.version }}" = "15" -o "${{ matrix.version }}" = "16" ] && [ "${{ matrix.os }}" = "windows-latest" ]; then
+              which clang-format
+              which clang-tidy
+          elif [ "${{ matrix.os }}" = "ubuntu-latest" ]; then
+              which "clang-format-${{ matrix.version }}"
+              which "clang-tidy-${{ matrix.version }}"
           fi
 
       - name: Check clang-tools on Windows
         if: matrix.os == 'windows-latest'
         shell: bash
         run: |
-          if [ "${{ matrix.version }}" = "15" ]; then
-            clang-format.exe --version
-            clang-tidy.exe --version
-          else
-            clang-format-${{ matrix.version }}.exe --version
-            clang-tidy-${{ matrix.version }}.exe --version
-          fi
+          case "${{ matrix.version }}" in
+            15|16)
```

Comment:
Does this work? I think you are comparing a string to an integer.

Does it matter how the binary is invoked?

Reply:
> sorry I'm not sure what do you mean

Why is it important to call `clang-tidy-15 --version` instead of the fallback `clang-tidy --version`?

Reply:
I suppose it matters because of how this pkg names the binaries/symlinks that it installs.